main/src/org/jetbrains/bsp/bazel/BazelBspServerConfig.java

```diff
@@ -0,0 +1,35 @@
+
+package org.jetbrains.bsp.bazel;
+
+public class BazelBspServerConfig {
+
+    private String pathToBazel = null;
+    private String projectPath = "//...";
+
+    public BazelBspServerConfig(String pathToBazel) {
+        this.pathToBazel = pathToBazel;
+    }
+
+    static public BazelBspServerConfig from(String[] args) {
+        if(args.length == 0) { throw new IllegalArgumentException("Configuration can't be built without any parameters"); }
+        BazelBspServerConfig config = new BazelBspServerConfig(args[0]);
+        if(args.length == 2) {
+            config.setProjectPath(args[1]);
```

Comment:
I think this may be too limitating for the usages of bazel. For instance we might be using the following structure:
- src
  * proj1
  * proj2
  * proj3
And we may want to import proj1 and proj3 but not proj1. This is a simple example but that can become much more complicated

Reply:
That is a really interesting point (I guess you meat `but not proj2`) , but in this specific case, would you want to import proj1 and proj3 because there is a dependency between the two ? Or it because you would like to work on them at the same time ? (I rarely open my whole monorepo into a single editor, I tend to open project by project and spawn a different server for each one of them, this is why I'm asking).
Assuming we want to keep this approach:
1. Should we provide the ability to the user to load a list of target instead of a single one ?
2. Should we do something about source dependencies ?

Reply:
Let's say that it is because you want to work on both. I didn't think this would be an issue either but after some talk with people who frequently work in monorepo structures, we foud ou that this was a common necessity.
Regarding your questions:
1. I think having the ability to select the targets would be wonderful
2. Source dependencies will have to be indexed either way but that shouldn't require any more work into it after specifiying the dependencies, at least I don't think so

Reply:
It is now possible to use multiple targets when starting the server by specifying them using a comma as delimiter:

```sh 
java -classpath bazel-bsp-0.1.0-SNAPSHOT.jar org.jetbrains.bsp.bazel.server.Server /usr/local/bin/bazel //toy2,//toy,//lib/otherlib
```

Is there something else we should take into account ?

Reply:
maybe we should also be able to receive separated by a space, wdyt?

Reply:
I can implement it if you think that is a nice to have but I'm not a really big fan of that.
What we could support on the other hand is having (like in the Install) a real command line argument parsing in the server, this could allow us to have multiple `--target (-t)` targets flags when starting the server 

```
org.jetbrains.bsp.bazel.server.Server /usr/local/bin/bazel -t //toy2 -t //toy -t //lib/otherlib
```